webapp/channels/src/components/browse_channels/browse_channels.tsx

```diff
@@ -197,6 +197,10 @@ export default class BrowseChannels extends React.PureComponent<Props, State> {
                     }
 
                     if (data) {
+                        const channelIDsForMemberCount = data.map((channel: Channel) => channel.id);
```

Comment:
@hmhealey I also thought about adding throttling but I wasn't sure because there was no throttling on the search. Please let me know if that makes sense to you?

Reply:
That's definitely a good idea, but I think the `setTimeout` here is being used for that. It's a debounce instead of a throttle since the last `search` call is the one that actually makes a request, but I think that's better in this case